src/ingest-pipeline/misc/tools/survey.py

```diff
@@ -702,7 +702,7 @@ def create_dataset(self,
         ingest_url = ENDPOINTS[self.instance]['ingest_url']
         data = {"provider_info": provider_info,
                 "contains_human_genetic_sequences": contains_human_genetic_sequences,
-                "data_types": [assay_type],
```

Comment:
I'm a bit concerned here - I believe that Joel's code relies on the data_types field for the old style data, so if we were to process any old style data/create datasets, this will break the assay classifier since the data_types would be blank.

Reply:
@jswelling could you confirm whether my assumption here is valid or not?

Reply:
@jpuerto-psc @sunset666 yeah, I'm concerned about the logic at this point.  CEDAR-style input records will have a less-specific 'dataset_type' field rather than the completely specific 'assay_type' field, and the call from split_and_create to create_dataset doesn't seem to allow for this new structure.  I suspect create_dataset has to accept either an assay_type or a dataset_type, and needs to build the metadata record for the new dataset appropriately in either case.  Those records would look like old- and new- style dataset metadata.  If only new-style is meant to be supported, this code won't do it.

Reply:
@jswelling, @jpuerto-psc, we might need to setup a discussion around this since the approach here was to use the new softassay classifier to get the assay_type.

Reply:
Please loop me in too @sunset666 :)

Reply:
Yeah- I'm honestly having a hard time visualizing the logic flow.

Reply:
@jswelling - are you avail for our IEC dev meeting tomorrow? I'd like to discuss this then.

Reply:
As per 01/05, we can merge these changes into devel so that Sunset can test changes on DEV. We will continue this discussion on 01/11, but that should not block Sunset's testing.

Reply:
Always send data_types and dataset_type, but send an empty string for the data_types field for new CEDAR style datasets.

we need a mapping of old non-CEDAR style dataset information to new CEDAR dataset_type. 

Ex:
- ATACseq-bulk -> ATACseq